roxctl/netpol/connectivity/diff/netpol_test.go

```diff
@@ -21,22 +20,17 @@ type diffAnalyzeNetpolTestSuite struct {
 	suite.Suite
 }
 
-func (d *diffAnalyzeNetpolTestSuite) TestDiffAnalyzeNetpol() {
+func (d *diffAnalyzeNetpolTestSuite) TestValidDiffCommand() {
 	tmpOutFileName := d.T().TempDir() + "/out"
 	outFileTxt := tmpOutFileName + ".txt"
 	cases := []struct {
 		name                  string
 		inputFolderPath1      string
 		inputFolderPath2      string
-		strict                bool
-		stopOnFirstErr        bool
 		outFile               string
-		outputToFile          bool
-		outputFormat          string
+		runCommand            bool
```

Comment:
Please try to rewrite the tests without this parameter (or assume it must always be false).
If that makes a test case impossible, then it means that the test case belongs to
`TestAnalyzeDiffCommand`.

Reply:
done